.github/workflows/zip-release.yml

```diff
@@ -0,0 +1,28 @@
+name: Zip-Release
+on:
+  push:
+    branches:
+      - "main"
+  pull_request:
+
+jobs:
+  release:
+    if: github.ref == 'refs/heads/main'
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Zip logos for domjudge
+        run: cd ${{ github.workspace }}; mkdir out;cd organizations; for f in *; do cp "$f/logo.64.png" "../out/$f.64.png"; done; cd ../out; zip -r domjudge.zip *.png;
```

Comment:
DOMjudge, at least with the import script, also expects a folder per organization.

If you want to directly copy it into DOMjudge, it still uses `ID.png`

Reply:
added now. should have the correct format in the third created zip.